models/login_source.go

```diff
@@ -266,7 +303,7 @@ func CreateLoginSource(source *LoginSource) error {
 
 // LoginSources returns a slice of all login sources found in DB.
 func LoginSources() ([]*LoginSource, error) {
-	auths := make([]*LoginSource, 0, 5)
+	auths := make([]*LoginSource, 0, 6)
```

Comment:
this can be any value, since it will look for all the configured login sources (and so depends on the environment how many this will be)